okta/resource_okta_policy_password.go

```diff
@@ -126,6 +126,14 @@ func resourcePolicyPassword() *schema.Resource {
 				Description: "If a user should be informed when their account is locked.",
 				Default:     false,
 			},
+			"lockout_notifications": {
+				Type:        schema.TypeSet,
+				Optional:    true,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				MinItems:    1,
+				Description: "If e-mail should be sent when the user when their account gets locked.",
```

Comment:
nit: Grammar fix -> Change `If e-mail should be sent when the user` to  `If email should be sent to the user`

Reply:
Thanks! Will submit a commit soon.